openupgradelib/openupgrade.py

```diff
@@ -1056,12 +1060,16 @@ def wrapped_function(cr, version):
                 logger.info(
                     "%s: %s-migration script called with version %s" %
                     (module, stage, version))
+                # Set up useful default context values
+                context = context or {}
+                context['migrate_version'] = version and version or None
+                context['module_name'] = module and module or None
```

Comment:
It's entirely the maintainers' call. I have seen disagreement among maintainers about the single-purposeness of openupgradelib already in the past. This is definitely a question of "product strategy" - not my business, unless someone would ask me for my opinion.